x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.tsx

```diff
@@ -154,6 +158,20 @@ const validateFormValues = (values: MaybeImmutable<NewTrustedApp>): ValidationRe
             values: { row: index + 1 },
           })
         );
+      } else if (
+        !isPathValid({ os: values.os, field: entry.field, type: entry.type, value: entry.value })
+      ) {
+        // show soft warnings and thus allow entry
+        isValid = true;
+        addResultToValidation(
+          validation,
+          'entries',
+          'warnings',
+          i18n.translate('xpack.securitySolution.trustedapps.create.conditionFieldInvalidPathMsg', {
+            defaultMessage: '[{row}] Path may be formed incorrectly; verify path value',
```

Comment:
the `path` feels redundant to me. Suggestion (optional):

```suggestion
            defaultMessage: '[{row}] Path may be formed incorrectly; verify value',
```